src/plugin.js

```diff
@@ -5,22 +5,25 @@ import { isSocketIo } from './utils';
 import defaults from './defaults';
 
 /**
- * @param {Vue} Vue
+ * @param {Vue} app
  * @param {SocketIOClient} socket
  * @param {Object} obj
  * @return {Object}
  */
-function defineReactiveProperties(Vue, socket, obj) {
-  const configStore = new Vue({
-    data: () => ({
-      connected: false,
-    }),
-    computed: {
-      disconnected() {
-        return !this.connected;
-      },
-    },
-  });
+function defineReactiveProperties(app, socket, obj) {
+  // const configStore = app.mixin({
+  //   data: () => ({
+  //     connected: false,
+  //   }),
+  //   computed: {
+  //     disconnected() {
+  //       return !this.connected;
+  //     },
+  //   },
+  // });
+  const configStore = {
```

Comment:
Since Vue 3 provides reactive wrappers, I believe we can use them here instead so `connected` property would work as before:

```js
import { ref } from 'vue'
const connected = ref(false);
// use `connected.value` instead of `configStore.connected` later
```